templates/referent/territorial_council/members.html.twig

```diff
@@ -150,13 +150,20 @@
                                         </div>
                                     </td>
                                     <td>
+                                        {% set pcQualities = adherent.politicalCommitteeMembership ? adherent.politicalCommitteeMembership.qualityNames : [] %}
                                         {% for quality in membership.qualities %}
                                             <b>{{ ('territorial_council.membership.quality.' ~ quality.name)|trans }}</b> {{ quality.zone }}
 
                                             {% if adherent.findMandatesForQuality(quality.name, true)|length %}
                                                 <span class="elected-tag elected-tag--small text--on-yellow text--bold b__nudge--left-nano">désigné{{ adherent.isFemale ? 'e' }}</span>
                                             {% endif %}
                                             <br />
+                                            {% if pcQualities %}
+                                                {% set pcQualities = pcQualities|filter((value) => value != quality.name) %}
```

Comment:
is it normal that this variable can be modified multiple times before getting used ?

Reply:
Yes, I want to display only PC qualities which don't correspond to TC qualities, that's why they are removed from the variable.

Reply:
what about improving it a little bit: 
```twig 
{% set pcQualities = adherent.politicalCommitteeMembership ? adherent.politicalCommitteeMembership.qualityNames : [] %}

{% for quality in membership.qualities %}
  // ...
{% endfor %}

{% for pcQuality in pcQualities|filter(quality => quality not in membership.qualities) %}
  // ...
{% endfor %}

```